### DISCUSSION TITLE
Allow choosing the language for the pluralizer.

### DISCUSSION THREAD

**Discussion by gerardp** [misterdebug/crud-generator-laravel#11] [Ideas]
Pluralizer::useLanguage('spanish');

Docs:
https://laravel.com/docs/9.x/localization#pluralization-language

**Comment by misterdebug** [comment]
I18n is not planned for now but you can change text in your default-theme folder :)

**Comment by gerardp** [comment]
Laravel, with its Plural or Singular functions, takes into account the language. For example, by default if you define a CRUD as categories it generates the model and controller as 'category' but if you define the pluralizer as 'spanish' it generates it as 'categoria'.

**Comment by misterdebug** [comment]
Do you use spanish in your file name or table name? 🤔

**Comment by gerardp** [comment]
Yes.

**Comment by misterdebug** [comment]
I've converted into discussion. If other people are interested in
